automatic/nexus-repository/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,210 @@
+$ErrorActionPreference = 'Stop'
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$version = '[[VERSION]]'
+$url64 = '[[URL]]'
+$checksum64 = '[[CHECKSUM]]'
+$checksumType64 = 'SHA256'
+$validExitCodes = @(0)
+Import-Module (Join-Path $toolsDir 'supportingfunctions.psm1')
+
+$name = $env:Computername
```

Comment:
This is overly complex here. There's nothing inside Nexus after installation from this package that needs to be this strict. Setting `$hostname = 'localhost'` will be sufficient, and should get around issues with the verifier